lib/buffer.js

```diff
@@ -36,7 +36,10 @@ function Buffer(arg) {
     // If less than zero, or NaN.
     if (arg < 0 || arg !== arg)
       arg = 0;
-    return allocate(arg);
+    const buffer = allocate(arg);
+    if (arguments[1])
```

Comment:
It was that way and I requested it change. Because the second argument is used for multiple signatures, so the named variable would conflict between the two.